src/main/resources/swagger-ui/yaml/flat.yaml

```diff
@@ -94,6 +94,31 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+    delete:
+      summary: Remove repository
+      operationId: removeRepo
+      tags:
+        - repository
+      parameters:
+        - name: rname
+          in: path
+          required: true
+          description: Name of the repository
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Remove a repository with name {rname}
+        '400':
+          description: Wrong repository name
+        '409':
```

Comment:
Error code 409, but in the `com.artipie.api.RepositoryRestFlatTest#deleteRepoNotfound` you do check `NOT_FOUND_404 = 404`.
Where is the mistake?

Reply:
It was error in yaml.